tests/integration/test_gardenlinux.py

```diff
@@ -285,7 +285,7 @@ def test_docker(client):
     if exit_code != 0:
         (journal_rc, output, error) = client.execute_command("sudo journactl --no-pager -xu docker.service")
     assert exit_code == 0, f"no {error=} expected"
-    (exit_code, output, error) = client.execute_command("sudo docker run --rm  eu.gcr.io/gardenlinux/gardenlinux:184.0 sh -c 'echo from container'")
+    (exit_code, output, error) = client.execute_command("sudo podman run --rm  eu.gcr.io/gardenlinux/gardenlinux:184.0 sh -c 'echo from container'")
```

Comment:
Do not replace Docker with Podman here, these tests run inside the finished image which still contain Docker and no PodMan.